### PR TITLE
Replace bootstrap.js with vanilla js

### PR DESCRIPTION
**Pull request by xPaw**
This removed the animation (it was jquery), but we can animate with CSS if we want to keep it.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
 					</button>
 				</li>
 				<li class="hidden-sm hidden-md hidden-lg">
-					<button id="menu-btn" title="Menu" class="collapsed" data-toggle="collapse" data-target="#mobile-menu" aria-expanded="false">
+					<button id="menu-btn" title="Menu">
```
Comment (astorije) [inline _layouts/default.html]: Can you port the `aria-expanded` behavior over as well?

**Comment by astorije** [comment]
> we can animate with CSS if we want to keep it

Yeah I'd rather keep the animation, it's a lot more intuitive (among other things because every other website out there has it).

File: _layouts/default.html (hunk 1 above)
Comment (xPaw) [reply _layouts/default.html]: I thought about it, but what mobile browser/software supports these?

Comment (astorije) [reply _layouts/default.html]: Not sure, it's an ARIA attribute, so probably VoiceOver and/or JAWS and/or other assistive technology.

File: css/style.css [hunk 2]
```diff
@@ -356,6 +356,13 @@ img {
 	width: 93px;
 }
 
+#mobile-menu {
+	display: block;
+	overflow: hidden;
+	max-height: 0;
+	transition: max-height 0.2s ease;
```
Comment (astorije) [inline css/style.css]:
```suggestion
	transition: max-height 0.35s ease;
```

Making this same as Bootstrap, which is less harsh on phone I find. nbd but trusting the giants here :D

**Pull request merged**